ios/11/_sentry_arm64.crash

```diff
@@ -0,0 +1,36 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BAD_ACCESS (SIGBUS)
+Exception Codes: BUS_NOOP at 0x10469c000
+Crashed Thread: 0
+
+Application Specific Information:
+Attempted to dereference garbage pointer 0x10469c000.
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x1000cfd24         -[CRLCrashGarbage crash] (CRLCrashGarbage.m:52)
+1   CrashLibiOS                     0x1000cfd20         -[CRLCrashGarbage crash] (CRLCrashGarbage.m:41)
```

Comment:
Reporting the same method twice with different lines and the method doesn't call itself recursively results in this being not complete. So this frame should be marked as invalid frame and the result for 64bit here is incomplete.